pulsar-functions/localrun/src/main/java/org/apache/pulsar/functions/LocalRunner.java

```diff
@@ -345,23 +345,38 @@ public void start(boolean blocking) throws Exception {
                         functionDetails = FunctionConfigUtils.convert(
                                 functionConfig,
                                 FunctionConfigUtils.validateJavaFunction(functionConfig, builtInFunctionClassLoader));
-                    } else if (Utils.isFunctionPackageUrlSupported(userCodeFile)) {
+                    } else if (userCodeFile != null && Utils.isFunctionPackageUrlSupported(userCodeFile)) {
```

Comment:
`isFunctionPackageUrlSupported ` is already defensive against null userCodeFile.
IDEA just doesn't see it. I can remove the null-check and it will show the IDEA warning.
WDYT ?